### DISCUSSION TITLE
Feature Request: Please add text shadow

### DISCUSSION THREAD

**Discussion by satyahbp** [tailwindlabs/tailwindcss#12972] [Ideas]
Please add the feature of text shadow in tailwindcss. It will be really helpful.
I tried drop shadow, but we can't change color of drop shadow. So it'll be great if we have the feature of text-shadow with all the colors!

**Comment by adamwathan** [comment]
Hey! Been wanting to add text-shadow for years actually but have never managed to decide on the right set of default values to include. Can you share an example of the type of design you're trying to achieve to help us figure out the right things to include out of the box?

**Reply by satyahbp** [reply]
Here's a screenshot of the type of design I am working on currently.
![image](https://github.com/tailwindlabs/tailwindcss/assets/26854500/bbf10258-edb0-42b4-8992-119f58784f65)

If you see, the text seems to be having a tiny bit of bright-aura, because of the text shadow that I have put. And I wanted the text shadow to be a lighter shade of gray. But I couldn't find that shade in Drop Shadow of tailwind, also in the documentation I saw that I have to customize it in config file to get that shade. 
But it was easier to write in plain CSS than to configure it in tailwind and get it:
`#main-title{
    text-shadow: 0px 0px 5px rgba(105, 105, 105, 0.486);
}`

Hence, if we have the text-shadow property with the same colors that we have for box shadow, it will be a great help!

I hope you find this explanation useful!
